WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -262,6 +266,9 @@ private extension TotalsView {
                         isShowingPaymentsButtonSpacing = true
                     }
                 }
+            if isShowingReceiptsNotEligibleBanner {
+                POSReceiptEligibilityBanner(isVisible: $isShowingReceiptsNotEligibleBanner)
```

Comment:
I had to play with this one a bit, as the ZStack was making the notice to not show on landscape, but do on portrait. Fixed it on https://github.com/woocommerce/woocommerce-ios/pull/14587/commits/b9f96bc0fe2dba5b33f2f3ffc3fe3dad897c8683

![Simulator Screenshot - iPad mini (A17 Pro) - 2024-12-05 at 09 58 15](https://github.com/user-attachments/assets/e8aa9970-9063-4a80-8235-498ca3f6f6f0)

It shows a bit hidden for large accessibility sizes due to the floating connection button, but I think this is fine for now:

![Simulator Screenshot - iPad mini (A17 Pro) - 2024-12-05 at 10 09 11](https://github.com/user-attachments/assets/f466fcc4-b5d5-4294-b758-5e168a42f0ea)

Reply:
> Also, only the receipt button title animates while other views just snap. If we keep a solution without Z-stack, then it would be nice to fix this.

Good eye! Do you think this is some sort of implicit animation coming from the parent? I could only address this by being explicit in the Text component about disabling it, fixed on https://github.com/woocommerce/woocommerce-ios/pull/14587/commits/1f7cc976764ab490a9dfd7920717df7ee40cd1ac

Reply:
> Good eye! Do you think this is some sort of implicit animation coming from the parent? 

Likely, but the Receipt button can continue animating, just together as one element rather than outline and title separately. I don't know fix helped and I can still see issues.

---------

However, the whole view continues to move with the toast, do you reproduce it? It shouldn't happen with `ZStack`. 

Also, wrap `isShowingReceiptsNotEligibleBanner` in `withAnimation` block or add `.animation(.default, value: isShowingReceiptsNotEligibleBanner)` to have a proper animation when this value changes.

https://github.com/user-attachments/assets/fccc48ac-375a-4100-8956-978b2f3df29d